spec/lang/syntax.md

```diff
@@ -431,7 +431,7 @@ Finally, the general structure of programs and functions:
 /// The internal representations of these types do not matter.
 pub struct FnName(pub libspecr::Name);
 pub struct GlobalName(pub libspecr::Name);
-pub struct VTableName(pub libspecr::Name);
+pub struct VTableName(pub TraitName, pub libspecr::Name);
```

Comment:
This is very odd. A name shouldn't usually contain another name. What is happening here?

Reply:
The reason is that in the builder API the constant both needs the vtable name and trait name, which is redundant.
But yes, it is weird.

Reply:
The builder API should never affect what happens in `spec/`.